polkadot/node/subsystem-bench/grafana/cpu-profiling.json

```diff
@@ -0,0 +1,70 @@
+{
```

Comment:
Is this a real time flamegraph or how does it work ? I would love to see a screenshot :)

Reply:
Added to the description